src/mesh/mesh.cpp

```diff
@@ -582,14 +582,14 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, RestartReader &rr,
   // the file is already open and the pointer is set to after <par_end>
 
   // All ranks read HDF file
-  nbnew = rr.GetAttr<int>("Mesh", "nbnew");
-  nbdel = rr.GetAttr<int>("Mesh", "nbdel");
-  nbtotal = rr.GetAttr<int>("Mesh", "nbtotal");
-  root_level = rr.GetAttr<int32_t>("Mesh", "rootLevel");
+  nbnew = rr.GetAttr<int32_t>("Info", "NBNew");
+  nbdel = rr.GetAttr<int32_t>("Info", "NBDel");
+  nbtotal = rr.GetAttr<int32_t>("Info", "NumMeshBlocks");
+  root_level = rr.GetAttr<int32_t>("Info", "RootLevel");
 
-  auto bc = rr.ReadAttr1DReal("Mesh", "bc");
+  auto bc = rr.GetAttrVec<std::string>("Info", "BoundaryConditions");
```

Comment:
can bc be a const reference?

Reply:
I'll make it const, but reference seems confusing, because `GetAttrVec` returns a value, not a reference. It can't return a reference because it just creates the vector and returns it, it doesn't store the vector.

Reply:
Sounds good!